docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -202,37 +201,34 @@ Typically, you need to back-fill the recording rules when you are using a long r
 and you upgraded to Scylla Monitoring 3.8 about three months ago.
 
 If you open the Overview dashboard and look at your entire retention time (in our example 1 year) you will see that while most of the graphs do
-show the data, the latency graphs have a missing period, in our example - from the entire year, the latency graph will only show the last three months.
-
-That nine months gap (12 months minus 3) is what we want to fill with back-filling.
+display the data, the latency graphs are missing a period of time.  The latency graph will only show the last three months in our example from the entire year. That nine months gap (12 months minus 3) is what we want to fill with back-filling.
```

Comment:
I find this confusing, all those durations are a real world example, it's not always going to be that.

Reply:
@amnonh I would leave it as is, because the entire "Determine the end time" section is an example - and it is emphasized in the first paragraph of that section ("for example, you have a year of retention data") and in the following ones ("in our example 1 year"). For sure, it's clearer than the previous version.
If we want to make it clearer, the content reorganization would help:


Typically, you need to back-fill the recording rules when using a long retention period.

Example:

You have a year of retention data, and you upgraded to ScyllaDB Monitoring 3.8 about three months ago. If you open the Overview dashboard and look at your entire retention time (one year), you will see that while most of the graphs do display the data, the latency graphs are missing a period of time. The latency graph will only show the last three months from the entire year. That nine months gap (12 months minus 3) is what we want to fill with back-filling.